include/fmt/core.h

```diff
@@ -1480,28 +1480,30 @@ template <typename OutputIt, typename Char> class basic_format_context {
    Constructs a ``basic_format_context`` object. References to the arguments are
    stored in the object so make sure they have appropriate lifetimes.
    */
-  basic_format_context(OutputIt out,
-                       basic_format_args<basic_format_context> ctx_args,
-                       detail::locale_ref loc = detail::locale_ref())
+  constexpr basic_format_context(
+      OutputIt out, basic_format_args<basic_format_context> ctx_args,
+      detail::locale_ref loc = detail::locale_ref())
       : out_(out), args_(ctx_args), loc_(loc) {}
 
-  format_arg arg(int id) const { return args_.get(id); }
-  format_arg arg(basic_string_view<char_type> name) { return args_.get(name); }
+  constexpr format_arg arg(int id) const { return args_.get(id); }
+  FMT_CONSTEXPR format_arg arg(basic_string_view<char_type> name) {
```

Comment:
Here and below: can we use `constexpr` instead of `FMT_CONSTEXPR`?

Reply:
Replacing `FMT_CONSTEXPR` with `constexpr` here (as here https://github.com/alexezeder/fmt/commit/1901008dd977eb29f90f6e1fbb01abf4a47f6d0b) leads to these failed actions:

- Linux - https://github.com/alexezeder/fmt/actions/runs/422139689
- MacOS - https://github.com/alexezeder/fmt/actions/runs/422139686

TLDR:
```
/home/runner/work/fmt/fmt/include/fmt/core.h:1489:24: error: 'constexpr' non-static member function will not be implicitly 'const' in C++14; add 'const' to avoid a change in behavior [-Werror,-Wconstexpr-not-const]
  constexpr format_arg arg(basic_string_view<char_type> name) {
                       ^
                                                              const
/home/runner/work/fmt/fmt/include/fmt/core.h:1495:35: error: 'constexpr' non-static member function will not be implicitly 'const' in C++14; add 'const' to avoid a change in behavior [-Werror,-Wconstexpr-not-const]
  constexpr detail::error_handler error_handler() { return {}; }
                                  ^
                                                  const
/home/runner/work/fmt/fmt/include/fmt/core.h:1499:22: error: 'constexpr' non-static member function will not be implicitly 'const' in C++14; add 'const' to avoid a change in behavior [-Werror,-Wconstexpr-not-const]
  constexpr iterator out() { return out_; }
                     ^
                           const
/home/runner/work/fmt/fmt/include/fmt/core.h:1506:32: error: 'constexpr' non-static member function will not be implicitly 'const' in C++14; add 'const' to avoid a change in behavior [-Werror,-Wconstexpr-not-const]
  constexpr detail::locale_ref locale() { return loc_; }
                               ^
                                        const
```